tensorflow/core/kernels/sparse_matmul_op.cc

```diff
@@ -1001,14 +1001,26 @@ class SparseMatMulOp : public OpKernel {
       // TODO(agarwal): multi-thread the conversions from bfloat16 to float.
       if (std::is_same<TL, bfloat16>::value) {
         a_float.reset(new Tensor(DT_FLOAT, a.shape()));
+#ifdef PLATFORM_GOOGLE
```

Comment:
We'd love to have this speedup at google to :) any reason to put it under ifdef?

Reply:
Thanks for the review.
Thanks because I have made a similar PR like this one before: https://github.com/tensorflow/tensorflow/pull/34212/files without `ifdef PLATFORM_GOOGLE` 
After several days, I find  `ifdef PLATFORM_GOOGLE`  has been added back(https://github.com/tensorflow/tensorflow/blob/1549473a2e9e50ed9d3c751f25eaf7ee6db180d4/tensorflow/core/kernels/resize_nearest_neighbor_op.cc#L145-L196). So  `ifdef PLATFORM_GOOGLE`  it will not use the eigen parallel version.

 Do you have any idea about exactly what this macro is controlling?